packages/react/src/River/RiverBreakout.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {Ref, forwardRef} from 'react'
+import clsx from 'clsx'
+import {RiverProps} from '.'
+import {useAnimation} from '../animation'
+import styles from './River.module.css'
+import {Heading} from '../Heading'
+
+type RiverBreakoutProps = {a11yHeading: string; a11yHeadingTag?: 'h2' | 'h3'} & Omit<
+  RiverProps,
+  'align' | 'imageTextRatio'
+>
+
+function findEmInChildren(children: React.ReactNode): boolean {
+  if (!children) {
+    return false
+  }
+
+  if (Array.isArray(children)) {
+    for (const child of children) {
+      if (findEmInChildren(child)) {
+        return true
+      }
+    }
+  } else if (React.isValidElement(children)) {
+    if (children.type === 'em') {
+      return true
+    }
+
+    if (children.props && children.props.children) {
+      if (findEmInChildren(children.props.children)) {
+        return true
+      }
+    }
+  }
+
+  return false
+}
+
+export const RiverBreakout = forwardRef(
+  (
+    {animate, a11yHeading, a11yHeadingTag = 'h2', className, children, style, ...rest}: RiverBreakoutProps,
```

Comment:
Not sure if I missed the API exploration for this, sorry if I did..

We typically use children to replace props like a11yHeading and a11yHeadingTag.

Could we, for example do something like this:

```jsx
<RiverBreakout>
  <RiverBreakout.Heading as="h2">Accelerate workflows<RiverBreakout.Heading>
  <RiverBreakout.Visual>
    <PlaceholderImage />
  </RiverBreakout.Visual>
  <RiverBreakout.Content trailingComponent={TrailingTimeline}>
    <Text>
      Accelerate your workflows and scale your business fast with access to millions of open source projects on
      GitHub, the largest source code host.
    </Text>
    <Link href="#">Call to action</Link>
  </RiverBreakout.Content>
</RiverBreakout>
```

☝️ differences are:

- `Visual` and `Content` are namespaced to RiverBreakout, but under hood share the same implementation as River
- Composable `Heading` which is a more familiar API for our users and allows props like `as` to work the same as everywhere else.